holidays/countries/thailand.py

```diff
@@ -682,10 +680,10 @@ def _populate_workday_holidays(self):
 
         # วันลอยกระทง
         # Status: In-Use.
+        # Started in 1941.
```

Comment:
```suggestion
```
This is just a cut-off date due to the Thai LuniSolar calendar support range, not its start date.

Reply:
Yes, but I consider this attribute ( `start_year`) as just a technical, not as "start" in a juridical sense. In fact, "just a cut-off".